### PR TITLE
Story1

### PR DESCRIPTION
**Pull request by danicatt**
As ready as I can be!

### CODE REVIEW

**Review by mporam — CHANGES_REQUESTED** [review]
difficulty pressing the nav links on mobile, consider spacing it out from the social icons

the about me pic isnt centralised

projects are not centralised on mobile

project images are not circles

**Comment by danicatt** [comment]
Made changes requested by Mike.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<link rel="stylesheet" href="task-header.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+
```
Comment (soneillcode) [inline index.html:7]: Worth sorting out textmate for indentation and spacing

File: index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+			</div>
+		</section>
+		
+		<section class="contactme"> 
+			<h2 id="contactdetails">Prefer e-mail? Drop me a line...</h2>
+			<h3> danicatterill@gmail.com </h3>
+			<a href="#top"><img src="toparrow.png" class="back2top" alt="Back to Top Button"/></a>
+		</section>
+	</body>
+</html>
```
Comment (soneillcode) [inline index.html:73]: This is missing a new line at the end of the file. It's sounds odd but it's important that all files have a new line at the end

**Review by soneillcode — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (danicatt) [reply index.html:7]: Fixed indentation on CSS and html with tabstospaces.com

File: index.html (hunk 2 above)
Comment (danicatt) [reply index.html:73]: Added a new line.

**Pull request closed without merging**